ReadMe.md

```diff
@@ -13,7 +13,7 @@ Command line
 
 >> oblique-strategies #prints random card
 
->> oblique-strategies help #prints help message
+>> oblique-strategies --help #prints help message
```

Comment:
could we add an example `-e` as well? 

The react site you have is really cool. Feel free to link it in here as well

Reply:
I added an example of `-e` is that what you mean? (see `--help`)